patches/chrome-browser-resources-settings-appearance_page-appearance_page.html.patch

```diff
@@ -1,21 +1,20 @@
 diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.html b/chrome/browser/resources/settings/appearance_page/appearance_page.html
-index fc68a42c1a6e349bbbba7170506cac1f250b25de..b2e30e892ce0bda3131aa9eef2a51aad41e09f57 100644
+index fc68a42c1a6e349bbbba7170506cac1f250b25de..10e24bcc5ce584531ef367a4a94af9766f12fe7f 100644
 --- a/chrome/browser/resources/settings/appearance_page/appearance_page.html
 +++ b/chrome/browser/resources/settings/appearance_page/appearance_page.html
-@@ -21,6 +21,12 @@
+@@ -21,6 +21,11 @@
  <link rel="import" href="../settings_vars_css.html">
  <link rel="import" href="home_url_input.html">
  
 +<if expr="not _google_chrome">
-+<link rel="import" href="../brave_appearance_page/brave_appearance_page.html">
-+<link rel="import" href="../default_brave_shields_page/default_brave_shields_page.html">
-+<link rel="import" href="../social_blocking_page/social_blocking_page.html">
++  <link rel="import" href="../brave_appearance_page/brave_theme.html">
```

Comment:
it doesn't seem like we should ever have more than one import because can't we can always create a new file that imports both? `brave/browser/resources/settings/appearance_page.html` with imports for brave_theme and toolbar?

Reply:
cc: @petemill